tests/data/test_loader.py

```diff
@@ -1,10 +1,11 @@
-from ert_data import loader
-from tests.data.mocked_block_observation import MockedBlockObservation
+from unittest.mock import MagicMock, Mock
+
+import deprecation
 import pandas as pd
 import pytest
-import deprecation
+from data.mocked_block_observation import MockedBlockObservation
```

Comment:
This line gives error when running pytest locally
This seems to solve it:
`from mocked_block_observation import MockedBlockObservation`

Reply:
changed